rpc_jobs/build_gating_venv.yml

```diff
@@ -0,0 +1,45 @@
+- job:
+    name: Build-Gating-Venv
+    project-type: workflow
+    concurrent: false
+    properties:
+      - rpc-gating-github
+    triggers:
+      - github # triggered post merge, not on PR
+    parameters:
+      - rpc_gating_params
+    dsl: |
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.shared_slave(){
+        withCredentials(artifact_build.get_rpc_repo_creds()) {
+          sh """#!/bin/bash -xeu
+            # Tar venv and roles
+            pushd rpc-gating
+              SHA=\$(git rev-parse HEAD)
+            popd
+            archive="rpcgatingvenv_\${SHA}.tbz"
+            set +u
+            . .venv/bin/activate
+            virtualenv --relocatable .venv
+            deactivate
+            set -u
+            tar cjfp \$archive .venv rpc-gating/playbooks/roles
```

Comment:
Fair enough to store roles as artifacts but it's confusing to mix them together, for me.
I think these are two different kind of artifacts: venvs != ansible roles.
If everybody agrees on it, fine for me.